### ISSUE TITLE
The reproduction for semantic segmentation

### ISSUE THREAD

**Issue by smalllwater** [yzd-v/MGD#19]
I have been replicating your work MGD recently. First of all, thank you for your contribution to mask knowledge distillation, your work has been a great inspiration to me.
During the reproduction process, I tried to keep the environment configuration consistent with the original configuration, applying the same torch, torchvision, mmcv, mmseg versions. Also tried to change various parameters, including learning rate, BN ,init_student , use_logit. I also tried adjusting the pretrained parameter to init in backbone, or downloading pre-trained weights to a local device. However, no matter how I adjust, my replication results are still well below expectations, with a 10% mIoU gap.
Would you please give me some guidance or advice?

[20250401_163951.log](https://github.com/user-attachments/files/19568837/20250401_163951.log)
